scrapper.py

```diff
@@ -48,14 +67,23 @@ def prepare_environment(base_path):
     """
     Creates ASSETS_PATH folder if not created and removes existing folder
     """
-    pass
+    try:
+        os.rmdir('ASSETS_PATH')
```

Comment:
I think that you do not use it correctly. It is not a string, it is a constant that should be imported from https://github.com/fipl-hse/2021-2-level-ctlr/blob/main/constants.py